feature-libs/cart/import-export/components/import-to-cart/import-entries-dialog/import-entries-dialog.component.html

```diff
@@ -26,84 +27,17 @@
-        </p>
-        <cx-file-upload
-          [control]="form.get('file')"
-          [buttonText]="'importEntriesDialog.selectFile' | cxTranslate"
-          [allowedExtensions]="fileValidity?.allowedExtensions"
-          [maxSize]="fileValidity?.maxSize"
-          (update)="loadFile($event, form)"
-        ></cx-file-upload>
-        <div class="cx-import-entries-row">
-          <ng-container>
-            <label>
-              <span class="cx-import-entries-label label-content">
-                {{ 'importEntriesDialog.savedCartName' | cxTranslate }}
-              </span>
-              <input
-                [maxLength]="nameMaxLength"
-                class="form-control"
-                formControlName="name"
-                required
-                type="text"
-              />
-              <cx-form-errors [control]="form.get('name')"></cx-form-errors>
-            </label>
-          </ng-container>
-        </div>
+      <cx-import-entries-form
+        *ngIf="formState; else importSummary"
+        (submitEvent)="importProducts($event)"
+      ></cx-import-entries-form>
 
-        <div class="cx-import-entries-row">
-          <label>
-            <span class="cx-import-entries-label label-content">
-              {{ 'importEntriesDialog.savedCartDescription' | cxTranslate }}
-              <span class="cx-import-entries-label-optional">
-                ({{ 'importEntriesDialog.optional' | cxTranslate }})
-              </span></span
-            >
-            <textarea
-              [maxLength]="descriptionMaxLength"
-              class="form-control"
-              formControlName="description"
-              rows="5"
-            ></textarea>
-            <cx-form-errors
-              [control]="form.get('description')"
-            ></cx-form-errors>
-
-            <p class="cx-import-entries-input-hint">
-              {{
-                'importEntriesDialog.charactersLeft'
-                  | cxTranslate: { count: descriptionsCharacterLeft }
-              }}
-            </p>
-          </label>
-        </div>
-        <div class="cx-import-entries-footer">
-          <button
-            (click)="close('Close Import Products Dialog')"
-            [attr.aria-label]="'importEntriesDialog.close' | cxTranslate"
-            class="btn btn-action"
-            type="button"
-          >
-            {{ 'importEntriesDialog.cancel' | cxTranslate }}
-          </button>
-          <button
-            [attr.aria-label]="'importEntriesDialog.upload' | cxTranslate"
-            class="btn btn-primary"
-            type="button"
-            [disabled]="form.invalid"
-            (click)="importProducts()"
-          >
-            {{ 'importEntriesDialog.upload' | cxTranslate }}
-          </button>
-        </div>
-      </div>
-    </form>
+      <ng-template #importSummary>
+        <cx-import-entries-summary
+          [summary$]="summary$"
```

Comment:
Can we simply unwrap observable `summary$ | async as summary` and avoid passing observable in inputs?   Btw. then we also avoid having $ in the suffix of the input